.github/scripts/windows/test_task.bat

```diff
@@ -91,16 +92,16 @@ rem work-around for failing to dl(mysqli) with OPcache (https://github.com/php/p
 if "%OPCACHE%" equ "1" set OPCACHE_OPTS=%OPCACHE_OPTS% -d extension=mysqli
 
 rem prepare for enchant
-mkdir %~d0\usr\local\lib\enchant-2
```

Comment:
%~d0 apparently means current drive. I believe using C:\ will make this more readable